### PR TITLE
Added deprecating warning in sigmac with color

### PR DESCRIPTION
**Pull request by elhoim**
Modified Pipfile to add depency to termcolor.
But i don't have a clue what to do with the Pipfile.lock content or how to generate it.

Did that because a colleague used sigmac but did not read the warning in the README

### CODE REVIEW

**Comment by Neo23x0** [comment]
Careful. 

1. I don't want users to get such a warning in just any execution. Only when they see the command line help. Otherwise the output would be pretty annoying. (e.g. see PyMISP and the super annoying deprecation messages)  
3. We have to change the text. We cannot 'remove' it or deprecate it somewhen in 2023.

**Comment by Neo23x0** [comment]
Text: 

```
Sigmac will be deprecated by the end of 2022 in favour of sigma-cli and pySigma. Please stop contributing backends to this tool. Limited support is offered until the end of 2023, especially for backends that haven't been migrated yet.
```

**Comment by thomaspatzke** [comment]
Good text, should be adeed to the top of the help.

File: tools/sigma/sigmac.py [hunk 1]
```diff
@@ -201,6 +207,7 @@ def main():
         sys.exit(0)
     elif len(cmdargs.inputs) == 0:
         print("Nothing to do!")
+        print(deprecation_warning_message)
```
Comment (frack113) [inline tools/sigma/sigmac.py:210]: As it is only when some one use wihtout rule , there is no need for the "Please stop contributing backends to this tool" part.

Comment (thomaspatzke) [reply tools/sigma/sigmac.py:210]: I think it's good to have the message regarding contribution present here and add a link to pySigma and sigma-cli, such that people who want to use our contribute directly are pointed to the right direction.

Comment (frack113) [reply tools/sigma/sigmac.py:210]: OK , let go for this.

**Pull request merged**